metricbeat/schema/mapstriface/mapstriface.go

```diff
@@ -128,7 +128,6 @@ func StrFromNum(key string, opts ...schema.SchemaOption) schema.Conv {
 func toStr(key string, data map[string]interface{}) (interface{}, error) {
 	emptyIface, err := common.MapStr(data).GetValue(key)
 	if err != nil {
-		fmt.Println(err)
 		return "", fmt.Errorf("Key %s not found", key)
```

Comment:
@ruflin, we do return an error here. it can be left upto the consumer to decide if this needs to be logged or not. for instance I got this error when I had set the `Conv` to optionally extract a field out. It should be logging errors for such cases. that would just fill up my log file because there may be many fields that are set to optional. thought?